casepro/statistics/models.py

```diff
@@ -55,6 +56,8 @@ def encode_scope(*args):
             return 'partner:%d' % args[0].pk
         elif types == [Org, User]:
             return 'org:%d:user:%d' % (args[0].pk, args[1].pk)
+        elif types == [Case]:
+            return 'case:%d' % (args[0].pk,)
```

Comment:
I had `[Org, Case]` here before but removed it because I had no clear reason why that'd be necessary.

For what would we want to track `[Case, User]` ?